code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/AuthDao.java

```diff
@@ -18,7 +22,29 @@
 
     Account createAccount(String name, String kind, String externalId, String externalType);
 
+    Account createProject(String name, String kind);
```

Comment:
So looks like this parameter is not used in the createProject method. Can we remove it then?